### ISSUE TITLE
Button is only clickable on text label

### ISSUE THREAD

**Issue by sled** [at-point/mtl#5]
Possibly the simple form plugin for materializeCSS generates wrong markup, i.e form submit buttons are only clickable on the text label:

<img width="1430" alt="screen shot 2016-06-28 at 14 00 00" src="https://cloud.githubusercontent.com/assets/320112/16414569/a6ecdff2-3d38-11e6-813e-4cad1ea7c820.png">

**Comment by lwe** [comment]
In addition we should probably also move the simple_form initializer to the mtl repository, in addition any idea what is better:

- keep initializer in mtl, include / run during initilization phase via engine
- copy initializer for simple form to config/initializers using `./bin/rails g mtl:install`?

**Comment by lwe** [comment]
@marcopluess what is the proposed / correct markup for this button?

**Comment by marcopluess** [comment]
@lwe - pretty much like this:

```HTML
<a class="waves-effect waves-light btn">button</a>
<a class="waves-effect waves-light btn"><i class="material-icons left">cloud</i>button</a>
<a class="waves-effect waves-light btn"><i class="material-icons right">cloud</i>button</a>
```

or HTML

```HTML
<button class="btn waves-effect waves-light" type="submit" name="action">Submit
    <i class="material-icons right">send</i>
</button>
```

**Comment by lwe** [comment]
Documentation of fix in #7
